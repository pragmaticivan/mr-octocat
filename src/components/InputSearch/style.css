.input {
  border-radius: 3px;
  border: 1px solid #d1d5da;
  box-shadow: inset 0 1px 2px rgba(27, 31, 35, 0.075);
  color: #24292e;
  font-size: 14px;
  line-height: 20px;
  padding: 6px 8px;
  vertical-align: middle;
  width: 100%;
}

.input::-webkit-input-placeholder, .input::-moz-placeholder, .input:-ms-input-placeholder, .input:-moz-placeholder {
  color: #d1d5da;
  font-size: 15px;
}
