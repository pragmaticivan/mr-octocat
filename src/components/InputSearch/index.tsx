import React from "react";
import style from "./style.css";

export default class InputSearch extends React.Component {
  render() {
    return (
      <input
        className={style.input}
        placeholder="Find a repository..."
        type="text"
      />
    );
  }
}
