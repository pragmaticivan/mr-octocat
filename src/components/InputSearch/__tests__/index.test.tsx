import InputSearch from "../";
import React from "react";
import { shallow } from "enzyme";

describe("InputSearch", () => {
  it("renders the component", () => {
    const wrapper = shallow(<InputSearch />);
    expect(wrapper).toMatchSnapshot();
  });
});
