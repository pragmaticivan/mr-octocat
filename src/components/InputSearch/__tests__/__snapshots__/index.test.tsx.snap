// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputSearch renders the component 1`] = `ShallowWrapper {}`;
